### DISCUSSION TITLE
No more Hugging Face token?

### DISCUSSION THREAD

**Discussion by Maxadox** [TheLastBen/fast-stable-diffusion#1443] [Q&A] [answered]
How can the colab work with no token? And what is the benefit of having a Gradio log in ?
Thank you all in advance

**Comment by TheLastBen** [comment, marked as answer]
the v1.5 model doesn't require a token anymore, but if you're working with a private model that requires a token, you can create a file "token.txt" containing the hf token and put it in the folder "Fast-Dreambooth" in your gdrive and it will automatically use it.

Having a gradio login protects your link from being used by malicious people.
